### ISSUE TITLE
Dyno cluster behind Network LB

### ISSUE THREAD

**Issue by maksimlikharev** [Netflix/dynomite#656]
This is a question.

I suppose nothing prevents me to have cluster, let say 3 nodes with the same token range ( 3 copies ), behind LB and use resulting DNS as an access point.

Trying to understand if this is something sane to do, also could it be used as drop in Redis replacement,  assuming restricted commands availability from Dyno.

**Comment by smukil** [comment]
@maksimlikharev Yes this is something you can do assuming you're working within the supported command set in Dynomite, and if it's for the purpose of load balancing across replicas.

Note however, that we run our replicas across AZs (on AWS), so having a single load balancer to balance across AZs might add some latency, which is why we go with the smart client route (Dyno).

However, if the architecture you mentioned makes sense for your use-case, there's no reason why it shouldn't work.

**Comment by maksimlikharev** [comment]
AWS Network LB clams to be "rack" or in this case AZ aware.
My real usecase is to migrate existing Redis clients, that can be migrated, specifically on other runtimes ( js / lua / python ).

For one client I want to go even further, setup DM in muli region, and connect clients using LB per region, want to see if this will fly, attempt to have the same replicated data across all regions, assuming all usual things, latencies/eventual consistency/etc.

**Comment by kanagasabai-karunanithy** [comment]
@smukil There is any place where i can see step by step procedure to setup DYNO for my env. I have two racks in the same DC. each rack has 3 nodes. 

dc:rack1 -3 nodes
dc:rack2 - 3 nodes

i am stuck on setting up dyno clients for clients to hit dynomite. Kindly help me with step by step procedure for setting up Dyno.

**Comment by maksimlikharev** [comment]
@kkarun004c - have you checked: https://github.com/Netflix/dyno/wiki/Configuration

is it not enough?

I do understand that you have functioning dyno cluster and straggle with client ?

**Comment by kanagasabai-karunanithy** [comment]
Hi @maksimlikharev, Thanks for your reply, I have gone through this page "https://github.com/Netflix/dyno/" to set up Dyno, after that, 

I have gone through this page "https://github.com/Netflix/dyno/wiki/Configuration" it's not clear to me, means, I am not a java background person. 

I am looking, under which path what config file needs to be there and how to start Dyno and how clients cant connect to my DynomiteDB cluster through Dyno. ?

**Comment by maksimlikharev** [comment]
check if this makes sense for you: 

[dyno-example](https://github.com/maksimlikharev/dyno-example)

should be public repo, just a simple example.
